docs/index.md

```diff
@@ -1,5 +1,9 @@
 # Shopsys Framework Knowledge Base
 
+## What is new and how to upgrade
+* For step by step instuctions see [UPGRADE.md](../UPGRADE.md)
```

Comment:
I would prefer having the Changelog information first - the headline states "What is new and how to upgrade" and then the first information is about upgrading - it sounds weird to me